.jules/runs/cartographer_roadmap_design_1/decision.md

```
@@ -0,0 +1,22 @@
+# Decision
+
+## Option A (recommended)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)